near-fetch/src/lib.rs

```diff
@@ -253,18 +282,24 @@ impl From<Client> for JsonRpcClient {
     }
 }
 
-async fn fetch_tx_errs(result: &FinalExecutionOutcomeView) -> Vec<&TxExecutionError> {
+async fn fetch_tx_errs(result: &RpcTransactionResponse) -> Vec<TxExecutionError> {
     let mut failures = Vec::new();
-
-    if let FinalExecutionStatus::Failure(tx_err) = &result.status {
-        failures.push(tx_err);
+    let outcome = result
+        .final_execution_outcome
+        .as_ref()
+        .unwrap()
```

Comment:
What is this unwrapping? Is it definitely going to succeed?

Reply:
yeah, this is gonna fail on the case that the tx hasn't been finalized yet. Look at the next comment for how this should be fixed